onnx/common/ir.h

```diff
@@ -1250,5 +1300,23 @@ inline const_graph_node_list_iterator Node::reverseIterator() const {
   return iterator().reverse();
 }
 
+inline const use_list Value::uses() const {
```

Comment:
Maybe add some comments to explain what this function (`uses`) is?

Reply:
Thanks! Updated.